### ISSUE TITLE
Console window pops up when using pythonw

### ISSUE THREAD

**Issue by kynax** [MSLNZ/msl-loadlib#31]
When using pythonw.exe (instead of python.exe) to make a call, a console window pops up to run the server32-windows.exe process.

You can recreate the behavior using the attached code.
[msl_console.txt](https://github.com/MSLNZ/msl-loadlib/files/8134874/msl_console.txt)

When running under python.exe (python.exe msl_console.txt), you get the expected result of the console getting the path, then time twice.

When running under pythonw.exe (pythonw.exe msl_console.txt), the expected result should be that nothing is printed to the active console.
The observed behavior is that a console window pops up and flashes a few times to run the server32-windows.exe process.

This is an issue when developping GUI apps since the users will be bombarded with console windows when using the app
![msl_console](https://user-images.githubusercontent.com/817238/155585929-bf996011-85e0-420a-b29b-8e0964265598.png)
.

**Comment by jborbely** [comment]
Could you try experimenting with different [STARTUPINFO](https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/ns-processthreadsapi-startupinfow) flags to get the desired result.

Modify [this line](https://github.com/MSLNZ/msl-loadlib/blob/60f100221774e7c8bac067b50f427fd1d99d2552/msl/loadlib/client64.py#L198) in your ``..\site-packages\msl\loadlib\client64.py`` file to be
```python
        si = None
        if sys.executable.endswith('pythonw.exe'):
            si = subprocess.STARTUPINFO()
            si.dwFlags = subprocess.STARTF_USESHOWWINDOW
            si.wShowWindow = subprocess.SW_HIDE
        self._proc = subprocess.Popen(cmd, stderr=subprocess.PIPE, stdout=subprocess.PIPE, startupinfo=si)
```
The ``dwFlags`` and  ``wShowWindow`` values that I used will stop the console window from popping up, but there is still an icon flashing in the taskbar.

**Comment by kynax** [comment]
I think this is a good start, but it doesn't seem to have the desired effect in my environment.

I'm actually running a pyQT5 app so I guess that the example that I provided earlier isn't the best, since you'd normally only use pythonw.exe if you actually initialize a GUI yourself.

Here is another more completed recreation of what the call might look like on my end.
You might need to 'pip install pyqt5' to run it.
[msl_qt5.txt](https://github.com/MSLNZ/msl-loadlib/files/8156062/msl_qt5.txt)

**Comment by jborbely** [comment]
What is the desired effect that you are still hoping for that has not already been achieved with the STARTUPINFO change? Is it that the console that you execute the ``pythonw.exe`` command _flashes_ for about 2 seconds?

For example, running the following script with `pythonw.exe qt5_example.py` causes the console to initially flash for about 2 seconds, when the ``self._k = Kernel64()`` line is executed, but afterwards I cannot tell the difference between running the script with ``python.exe`` versus ``pythonw.exe``.

```python
# qt5_example.py

import sys
from PyQt5.QtWidgets import *
from msl.examples.loadlib import Kernel64

class Window(QWidget):

    def __init__(self):
        super(Window, self).__init__()

        self._k = Kernel64()

        button_alert = QPushButton('Alert')
        button_alert.clicked.connect(self.on_alert_clicked)

        button_update = QPushButton('Get time')
        button_update.clicked.connect(self.on_update_clicked)

        self._label = QLabel()

        layout = QVBoxLayout()
        layout.addWidget(button_alert)
        layout.addWidget(button_update)
        layout.addWidget(self._label)
        self.setLayout(layout)

    def on_alert_clicked(self):
        alert = QMessageBox()
        alert.setText(sys.executable + '\n' + str(self._k.get_local_time()))
        alert.exec_()

    def on_update_clicked(self):
        self._label.setText(f'<html>The current time is:<br>'
                            f'<b>{self._k.get_local_time()}</b></html>')

    def closeEvent(self, event):
        event.accept()
        self._k.shutdown_server32()


if __name__ == '__main__':
    app = QApplication([])
    window = Window()
    window.show()
    sys.exit(app.exec_())
```
I ran your ``pythonw.exe msl_qt5.txt`` example, but that too does not create a pop up console.

**Comment by jborbely** [comment]
Great! Thanks for tracking down a fix to this issue.

I have pushed your suggested fix in 784321ded9f208f5f56df2a04ceb591aed228dc6 (with a slight modification to also support a non-Windows OS and Python < 3.7).

Could you please upgrade the package from the `main` branch to check that this issue will be resolved in the next release and then let me know the result. To install from `main` you can run

```console
python -m pip install -U https://github.com/MSLNZ/msl-loadlib/archive/main.zip
```

**Comment by kynax** [comment]
I installed from the main branch and can confirm that it fixes the issue.

Thank you!
